assets/src/components/parts/janus-capi-iframe/ExternalActivity.tsx

```diff
@@ -869,6 +869,13 @@ const ExternalActivity: React.FC<PartComponentProps<CapiIframeModel>> = (props)
         key: baseKey,
         value,
       });
+      const typeOfValue = typeof value;
```

Comment:
This fixes the absorbance issue.

Note:- we can make this changes in ParseArray() but i did it here because in SIMs, they expect it as string instead of array object for multi-dimensional array.